functions.php

```diff
@@ -114,6 +114,27 @@ function insert_into_template(positioning_text, added_text, insert_before) {
 			wp_add_inline_script( 'jquery', $script );
 		}
 	}
+
+	if ( 'themes.php' === $hook || 'plugins.php' === $hook ) {
```

Comment:
@ronalfy 

Can you please change this line to:

`if ( 'themes.php' === $hook || 'plugins.php' === $hook || $hook === 'site-themes.php' ) {`

so that AJAX works on the `Network Admin > Sites > Edit > Themes` screen.

Reply:
Thanks. I'll do some revisions after the core meeting if we decide we want to do Ajax for enabling/disabling.

Reply:
while we no longer need the suggested `site-themes.php` added to the condition, we do need to avoid enqueueing the JS in multisite when on the sub-site plugins/themes screens, i.e., change the conditional to:

`if ( ( 'themes.php' === $hook || 'plugins.php' === $hook ) && ( ! is_multisite() || is_network_admin() ) ) {`

Reply:
Fixed and tested on single/multisite with https://github.com/WordPress/wp-autoupdates/pull/61/commits/ca4f685136122948baa8dabff8c00883fcc9862f

Just a side note, we may want to do the same with the CSS, but I believe that's out of the scope of this PR.

Reply:
yeah, I've been meaning to find the tine to refactor `wp_autoupdates_enqueues()` to simplify the multisite-related checks.  Will do that once this PR gets meged.